feature-libs/checkout/base/components/checkout-delivery-address/checkout-delivery-address.component.ts

```diff
@@ -89,6 +89,20 @@ export class CheckoutDeliveryAddressComponent implements OnInit {
       region = address.region.isocode + ', ';
     }
 
+    let numbers = '';
+    if (address.cellphone && address.phone) {
```

Comment:
This seems like an unnecessary amount of conditions. Could we please simplify this?